threedi_modelchecker/checks/raster.py

```diff
@@ -300,6 +300,20 @@ def description(self):
         return f"{self.column_name} has values {' and/or '.join(parts)} or is empty"
 
 
+class RasterCompressionUsedCheck(BaseRasterCheck):
+    """Checks whether compression was used for the raster"""
+
+    def is_valid(self, path: str, interface_cls: Type[RasterInterface]):
+        with interface_cls(path) as raster:
+            if not raster.is_valid_geotiff:
```

Comment:
Why is not being a valid geotiff sufficient for being compressed?

Reply:
is_valid_geotiff returns False if the dataset is empty (such as when the file doesn't exist) or is not a valid geotiff. It's used on every raster check (such as RasterHasOneBandCheck) which reads the raster in any way, to prevent any other code in the check from executing if the dataset doesn't meet the require standards. If it doesn't exist, or isn't a valid geotiff, RasterExistsCheck and/or RasterIsValidCheck will be raised elsewhere.

Reply:
Ok, you may want to summarize this in a short comment

Reply:
I've put the comment at the top of the file because of it happening in all the checks.